AutolayoutPractice/RemindersList/Views/AddReminderViewController.swift

```diff
@@ -83,7 +85,15 @@ class AddReminderViewController: UIViewController {
         delegate?.addReminder(title: titleTextField.text ?? "", description: descriptionTextView.text ?? "", dueDate: dueDatePicker.date)
         dismiss(animated: true)
     }
-    
+
+    @objc private func textFieldDidChange(_ textField: UITextField) {
+        if textField.text?.isEmpty == true {
+            saveButton.isEnabled = false
```

Comment:
This could be simplified to:

`saveButton.isEnabled = textField.text?.isEmpty != true`